packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md

```diff
@@ -3,26 +3,52 @@ id: Writing design guidelines for PatternFly
 section: UX writing
 ---
 
-Design guidelines walk through building design patterns with PatternFly. This page focuses on how to write clear and consistent design documentation. To learn how to contribute design guidelines, see our [contribution guide](/contribute/design/).
+Design guidelines walk through building design patterns with PatternFly. This page focuses on how to write clear and consistent design documentation. To learn how to contribute design guidelines, see our [contribution guide](https://github.com/patternfly/patternfly-org/wiki/Contributing-to-patternfly-org-for-designers).
 
-## Content architecture
+## Design guidelines template
+When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must be under one of the following headings.
 
-Use headings, hierarchy, formatting, and lists to organize your design guidelines into digestible sections.
+### Elements (H2)
+Introduce the component and list the primary parts that make up its structure. This includes specific icons, headers, fields, symbols, or anything else included in the component.
+Example:
+<img src="sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px" />
```

Comment:
This image is inside of another folder, so the path to the image will need to be updated
```suggestion
<img src="./img/sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px" />
```